### DISCUSSION TITLE
Where is pnet datalink documentation?

### DISCUSSION THREAD

**Discussion by gardc** [libpnet/libpnet#591] [Q&A] [answered]
Where is pnet datalink documentation? Can't find it on docs.rs.

**Comment by s4093120** [comment]
i have the same question.

**Comment by mrmonday** [comment, marked as answer]
You can find it here: https://docs.rs/pnet_datalink/latest/pnet_datalink/

It looks like this has been broken since `v0.30.0`...

**Reply by mrmonday** [reply]
The main documentation is fixed in the latest release `v0.32.0`.
